rootfs/entrypoint.sh

```diff
@@ -1,8 +1,14 @@
 #!/usr/bin/dumb-init /bin/sh
 source version.sh
-
+if [ -n "${SSH_KEY}" ]; then
+    echo "$SSH_KEY" > /id_rsa/custom_rsa
```

Comment:
There should be no reason to have an extra file on disk.

We've already defined `/id_rsa` as the custom to file to use, (a) why create another, and (b) why use a completely non-standard directory?